flujo/domain/interfaces.py

```
@@ -1,206 +1,147 @@
-        """Load data from storage. Returns None if not found."""
+        def error(self, message: str, *args: Any, **kwargs: Any) -> None:
+            logger.error(message, *args, **kwargs)
 
-    async def save(self, key: str, data: T) -> None:
-        """Save data to storage."""
+        def debug(self, message: str, *args: Any, **kwargs: Any) -> None:
+            logger.debug(message, *args, **kwargs)
 
-    async def delete(self, key: str) -> bool:
-        """Delete data from storage. Returns True if deleted."""
+        def span(self, name: str, *args: Any, **kwargs: Any) -> Any:
```

Ruff (0.14.7)

Unused method argument: `name`

(ARG002)